.github/workflows/android_build.yml

```diff
@@ -56,9 +56,13 @@ jobs:
     - run: cd mobile && ./ci/mac_ci_setup.sh --android
       if: steps.should_run.outputs.run_ci_job == 'true'
       name: 'Install dependencies'
-    - name: 'Start simulator'
+    - uses: nick-fields/retry@943e742917ac94714d2f408a0e8320f2d1fcafcd
       if: steps.should_run.outputs.run_ci_job == 'true'
-      run: cd mobile && ./ci/mac_start_emulator.sh
+      name: 'Start emulator'
+      with:
+        timeout_minutes: 10
```

Comment:
could we lower it? Not sure whether 5 is not too low but I think that we should be fine with something that's lower than 10. Maybe somewhere between 5 and 7?

Reply:
Took 4:41s here: https://github.com/envoyproxy/envoy/actions/runs/3905297829/jobs/6673475684

Let's try with 6.